PHC-Android/PHC/src/main/res/values/strings.xml

```diff
@@ -2,7 +2,7 @@
 <resources>
 
     <!-- App -->
-    <string name="app_name">  Project Homeless Connect</string>
+    <string name="app_name">   Project Homeless Connect</string>
```

Comment:
Apparently it is to fake the padding on the PHC logo (to the best of my knowledge at least)!